### DISCUSSION TITLE
How to load deepmd/npy data

### DISCUSSION THREAD

**Discussion by ZSHtju** [deepmodeling/dpdata#365] [Q&A] [answered]
I have a set of deepmd/npy data, and I want to load deepmd/npy as system or labeledsystem and write them into a POSCAR file. Anyone could give me an example?

**Comment by wanghan-iapcm** [comment, marked as answer]
```
data = LabeledSystem('/path/to/system', fmt='deepmd/npy')
data.to('vasp/poscar', 'POSCAR', frame_idx= index_you_need)
```

**Reply by ZSHtju** [reply]
Many thanks for the answers. It works well.
